gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreWrapper.java

```diff
@@ -229,57 +252,80 @@ public Key apply(@Nullable Entity input) {
 
         final Key[] keysArray = new Key[keys.size()];
         keys.toArray(keysArray);
+        dropTableInternal(keysArray);
+    }
+
+    void dropTableInternal(Key[] keysArray) {
+        if (keysArray.length > MAX_KEYS_PER_WRITE_REQUEST) {
+            int start = 0;
+            int end = MAX_KEYS_PER_WRITE_REQUEST;
+            while (true) {
+                final int length = end - start;
+                if (length <= 0) {
+                    return;
+                }
+                final Key[] keysSubarray = new Key[length];
+                System.arraycopy(keysArray, start, keysSubarray, 0, keysSubarray.length);
+                delete(keysSubarray);
 
-        delete(keysArray);
+                start = end;
+                end = min(MAX_KEYS_PER_WRITE_REQUEST, keysArray.length - end);
+            }
+        } else {
+            delete(keysArray);
+        }
     }
 
     /**
      * Starts a transaction.
-     *
+     * <p>
```

Comment:
Please remove extra `<p>` here and in similar cases.  We don't put those for an empty line.